js/src/jupyterlab-plugin.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
                             console.error(err);
                             reject(err);
                         },
-                        'jupyter-threejs'
+                        'jupyter-threejs-chunk'
```

Comment:
This change is surprising to me. Why do  you need `-chunk`?

Reply:
the chunk name was not allowed to be the same name as package/module/forgotwhatitwasname, as long as it was not `jupyter-threejs` it was fine.

Reply:
can confirm, have encountered this elsewhere